AspNetCore3ChildlifetimeScope/Dependencies/TransientRootDependency.cs

```diff
@@ -0,0 +1,10 @@
+namespace AspNetCore3ChildlifetimeScope.Dependencies
+{
+    public interface ISingletonRootDependency
```

Comment:
The name of the classes in here aren't reflected by the filename, which makes it a little hard to navigate. Ideally each class or interface, even empty ones, would be in the standard "one class/interface per file with matching name" so folks can easily see where things are without having to open all the files.

Reply:
Note if the interface isn't interesting... you could always not have an interface and register things `AsSelf`.

Reply:
The other examples use a Controllers/Properties/Services folder structure. It might be good to stick to that, even if it's not technically required, so the examples are all consistent with each other.